api_test.go

```diff
@@ -109,7 +109,7 @@ func TestSizeBytesSearchResult(t *testing.T) {
 		LineFragments: nil, // 48 bytes
 	}
 
-	var wantBytes uint64 = 724
+	var wantBytes uint64 = 740
```

Comment:
nit: update the comment on line 81 to be 

`Stats: Stats{}, // 144 bytes`

or `144 bytes + 0 bytes` to clarify empty reason